appserver/itest-tools/pom.xml

```diff
@@ -65,6 +83,7 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter-engine</artifactId>
+            <scope>test</scope>
```

Comment:
Here it must be a compile scope, not a test scope. I wonder why it was not specified here.